Venmo/src/main/java/com/braintreepayments/api/VenmoRequest.java

```diff
@@ -41,5 +47,31 @@ public boolean getShouldVault() {
     public String getProfileId() {
         return profileId;
     }
+
+    /**
+     * @return {@link VenmoPaymentMethodUsage} for the tokenized Venmo account: either multi-use or single use.
+     */
+    public @VenmoPaymentMethodUsage int getPaymentMethodUsage() {
+        return paymentMethodUsage;
+    }
+
+    String getPaymentMethodUsageAsString() {
+        switch (paymentMethodUsage) {
+            case VenmoPaymentMethodUsage.SINGLE_USE:
+                return "SINGLE_USE";
+            case VenmoPaymentMethodUsage.MULTI_USE:
+                return "MULTI_USE";
+            default:
+                return null;
+        }
+    }
+
+    /**
+     * Set {@link VenmoPaymentMethodUsage} for the tokenized Venmo account: either multi-use or single use.
+     * Defaults to {@link VenmoPaymentMethodUsage#MULTI_USE}
+     */
+    public void setPaymentMethodUsage(@VenmoPaymentMethodUsage int paymentMethodUsage) {
```

Comment:
We can remove this now that we have it in the constructor, right?

Reply:
We technically don't need the getter either may take that out for now since we can always add back.

Reply:
Actually will keep there are good docs on that property.